src/main/scala/piconot/parser.scala

```diff
@@ -0,0 +1,60 @@
+package piconot
+
```

Comment:
The following code looks very good there is a nice simplicity to your implementation. Very clean and readable code. I would have enjoyed a further description of how your definitions map to your language. I feel like for myself i struggled to understand how to change the existing picobot language into an external dsl so I went for a modified internal dsl language. I think you did a very nice job implementing an external dsl and the simplicity of it is very nice.